src/pages/dar_application/DataAccessRequestApplication.js

```diff
@@ -228,11 +227,19 @@ const DataAccessRequestApplication = (props) => {
   const init = useCallback(async () => {
     const { dataRequestId, collectionId } = props.match.params;
     let formData = {};
-    const researcher = await User.getMe();
-    const signingOfficials = await User.getSOsForCurrentUser();
+    if (props.researcher === undefined) {
+      const researcher = await User.getMe();
+      const signingOfficials = await User.getSOsForCurrentUser();
+      setResearcher(researcher);
+      setAllSigningOfficials(signingOfficials);
+    }
+    else {
+      const researcher = props.researcher;
+      // const signingOfficials = await User.getSOsForGivenUser(props.researcher.userId);
+      setResearcher(researcher);
+      setAllSigningOfficials(['Hello']);
```

Comment:
Do we actually need a list of SOs in this condition? I think if it's read-only, the list can be empty.